src/OAuth1/Service/Jira.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace OAuth\OAuth1\Service;
+
+use OAuth\Common\Http\Exception\TokenResponseException;
+use OAuth\OAuth1\Token\StdOAuth1Token;
+
+class Jira extends AbstractService
+{
+    protected $baseApiUri = 'http://localhost/rest/api/2/';
```

Comment:
For now it is not configurable. As its a single files modification, so "$baseApiUri" we can edit this variable from "Jira.php" directly.